flink-libraries/flink-python/pom.xml

```diff
@@ -48,6 +48,14 @@ under the License.
                     </archive>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.17</version><!--$NO-MVN-MAN-VER$-->
+                <configuration>
+                    <forkCount>1</forkCount>
```

Comment:
uhh...i think something broke otherwise. Let me try it real quick and see what happens.